server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+
+*Consent Required*
+
+If this is on, then users will get a consent page which asks the user if they grant access to that application.  It will also display the metadata that the client is interested in so that the user knows exactly what information the client is getting access to.
+If you have ever done a social login to Google, you'll often see a similar page.  {project_name} provides the same functionality.
+
+[[_access-type]]
+*Access Type*
+
+The type of OIDC client.
+
+_confidential_::
+  Confidential access type is for server-side clients that need to perform a browser login and require a client secret when they turn an access code into an access token,
+  (see https://tools.ietf.org/html/rfc6749#section-4.1.3[Access Token Request] in the OAuth 2.0 spec for more details). This type should be used for server-side applications.
+
+_public_::
+  Public access type is for client-side clients that need to perform a browser login. With a client-side application there is no way to keep a secret safe. Instead it is very important to restrict  access by configuring correct redirect URIs for the client.
+
+_bearer-only_::
+  Bearer-only access type means that the application only allows bearer token requests.
+  If this is turned on, this application cannot participate in browser logins.
+
+*Standard Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Authorization Code Flow>>.
+
+*Implicit Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Implicit Flow>>.
+
+*Direct Access Grants Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Direct Access Grants>>.
+
+*Root URL*
+
+If {project_name} uses any configured relative URLs, this value is prepended to them.
+
+*Valid Redirect URIs*
+
+This is a required field.  Enter in a URL pattern and click the + sign to add.  Click the - sign next to URLs you want to remove.
+Remember that you still have to click the `Save` button!
+Wildcards (*) are only allowed at the end of a URI, i.e. $$http://host.com/*$$
+
+You should take extra precautions when registering valid redirect URI patterns. If you make
+them too general you are vulnerable to attacks.  See <<_unspecific-redirect-uris, Threat Model Mitigation>> chapter
+for more information.
+
+*Base URL*
+
+If {project_name} needs to link to the client, this URL is used.
+
+*Admin URL*
+
+For {project_name} specific client adapters, this is the callback endpoint for the client.  The {project_name}
+server will use this URI to make callbacks like pushing revocation policies, performing backchannel logout, and other
+administrative operations.  For {project_name} servlet adapters, this can be the root URL of the servlet application.
+For more information see link:{adapterguide_link}[{adapterguide_name}].
+
+*Web Origins*
+
+This option centers around link:https://fetch.spec.whatwg.org/[CORS] which stands for Cross-Origin Resource Sharing.
```

Comment:
What do you mean by "centers around"? I would begin this description with what the option does. Are you specifying a value here, or turning it on and off? For example, if it is a binary, then the description should start thusly:
"When enabled, it ensures that servers handle Cross-Origin Resource Sharing (CORS) appropriately...."